### PR TITLE
5.x: add info about removed code

### PR DESCRIPTION
**Pull request by LordSimal**
Refs:
- https://github.com/cakephp/cakephp/pull/15599
- https://github.com/cakephp/cakephp/pull/15604
- https://github.com/cakephp/cakephp/pull/15605
- https://github.com/cakephp/cakephp/pull/15643
- https://github.com/cakephp/cakephp/pull/15608
- https://github.com/cakephp/cakephp/pull/15774
- https://github.com/cakephp/cakephp/pull/15839
- https://github.com/cakephp/cakephp/pull/16173
- https://github.com/cakephp/cakephp/pull/16345
- https://github.com/cakephp/cakephp/pull/16395
- https://github.com/cakephp/cakephp/pull/16426
- https://github.com/cakephp/cakephp/pull/16465
- https://github.com/cakephp/cakephp/pull/16495

I'd still like to go through the whole https://github.com/cakephp/cakephp/compare/4.x...5.x compare view but this at least adds some much needed information to the Cake5 migration guide.

### CODE REVIEW

File: en/appendices/5-0-migration-guide.rst [hunk 1]
```diff
@@ -49,6 +56,11 @@ Controller
   change can impact applications that use ``property_exists()`` on components.
 - The components' ``Controller.shutdown`` event callback has been renamed from
   ``shutdown`` to ``afterFilter`` to match the controller one. This makes the callbacks more consistent.
+- ``PaginatorComponent`` has been removed and should be replaced by using
```
Comment (ADmad) [inline en/appendices/5-0-migration-guide.rst]: In controllers one can just use paginate(), no need to use the paginator directly. Also the class is now NumericPaginator.

File: en/appendices/5-0-migration-guide.rst [hunk 2]
```diff
@@ -34,10 +34,17 @@ Cache
 - The ``Wincache`` engine was removed. The wincache extension is not supported
   on PHP 8.
 
+Auth
+-------
```
Comment (dereuromark) [inline en/appendices/5-0-migration-guide.rst]: Arent those wrong length?

Comment (LordSimal) [reply en/appendices/5-0-migration-guide.rst]: right.... I always forget that

File: en/appendices/5-0-migration-guide.rst [hunk 3]
```diff
@@ -135,6 +163,14 @@ ORM
   calling functions unrelated to the specific query type.
 - ``Table::_initializeSchema()`` has been removed and should be replaced by calling
   ``$this->getSchema()`` inside the ``initialize()`` method.
+- ``SaveOptionsBuilder`` has been removed. Use a normal array for options instead.
+
+Routing
+-------
+
+- Static calls to ``connect``, ``prefix``, ``scope`` and ``plugin`` have been removed and
```
Comment (ADmad) [inline en/appendices/5-0-migration-guide.rst]:
```suggestion
- Static methods ``connect()``, ``prefix()``, ``scope()`` and ``plugin()`` of the ``Router`` have been removed and
```

File: en/appendices/5-0-migration-guide.rst [hunk 4]
```diff
@@ -135,6 +163,14 @@ ORM
...
+  should be replaced by calling their non-static method variants via the RouteBuilder.
```
Comment (ADmad) [inline en/appendices/5-0-migration-guide.rst]:
```suggestion
  should be replaced by calling their non-static method variants via the ``RouteBuilder`` instance.
```

**Comment by markstory** [comment]
Awesome stuff :clap:

File: en/appendices/5-0-migration-guide.rst [hunk 5]
```diff
@@ -115,6 +138,11 @@ Log
 - Log engine config now uses ``null`` instead of ``false`` to disable scopes.
   So instead of ``'scopes' => false`` you need to use ``'scopes' => null`` in your log config.
 
+Mailer
+---
```
Comment (dereuromark) [inline en/appendices/5-0-migration-guide.rst:142]: Does this need more chars?

**Pull request merged**